basic-steps/src/main/resources/org/jenkinsci/plugins/workflow/steps/ArtifactArchiverStep/help-excludes.html

```diff
@@ -0,0 +1,4 @@
+<div>
+Exclude artifacts matching this <a href="https://ant.apache.org/manual/dirtasks.html#patterns">Ant style pattern</a>.
+Use a comma separated list to add more than one expression.
```

Comment:
Also hyphenate `Ant-style` and `comma-separated`. (Helps avoid “garden path” parsing errors by readers.)